source/arriving/setup_computer.rst

```diff
@@ -1,51 +1,167 @@
+Now is the time to open a bash terminal, also known as the command line. The terminal allows you to execute commands, manage files and folders, and automate tasks. You will use it to install all you need to complete the setup of your computer.
 
-4. You may want to install other tools that are often useful for people in our lab (see the :doc:`here <../intro_to/explore_software>` page for MI-brain, Ants, MRtrix, FSL and Freesurfer, etc.) 
+It is useful to learn some of the basics of the bash language to help you play around. Please refer to this cheat sheet on the bash language (*TODO add link to new section*).
 
-Getting set up on super computers
-"""""""""""""""""""""""""""""""""
+Before proceeding to the next steps, here are some useful additions you can make to your terminal "settings", by modifying what we call the *.bashrc*. First, open the .bashrc in VSCode by typing in the terminal :bash:`code ~/.bashrc`, then, copy-paste these lines at the end of it:
 
-The first use of a computing platform can be tricky but you'll get used to it. Please see the (:ref:`ref_heavy_computing`) tab for more information and for our first-use tutorial. If your goal is to use the computing platform to run Tractoflow, you will find instructions on the :ref:`ref_tractoflow` page. Else see the :ref:`ref_other_pipelines` page.
+.. code-block:: bash
 
-Working from home
-"""""""""""""""""
+    # Useful aliases
+    alias c='clear'
+    alias histg='history | grep'
+
+    # History auto-complete navigation
+    bind '"\e[5~": history-search-backward' # Press page-up button to go backward in history
+    bind '"\e[6~": history-search-forward' # Press page-down button to go forward in history
+
+Python
+""""""
+
+Although your Linux distribution most likely already includes a python distribution, it may be necessary to install additionnal python packages for software development. There are many alternatives for installing python. On Ubuntu, we can use the `deadsnakes <https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa>`_ repository to download and install a specific version:
+
+.. code-block:: bash
+
+        sudo add-apt-repository ppa:deadsnakes/ppa
+        sudo apt-get install python3.10-dev python3.10-venv python3.10-tk python3-pip
+        python3.10 -m pip install pip  # update pip (python package manager) to latest version
+
+Other python packages can be installed by replacing python3.X by a version of your choice.
+
+Virtual environments
+""""""""""""""""""""
+
+Virtual environments are a good way to separate the system's python installation from the python version you need for your work. It also allows you to have a precise set of python packages with specific versions. While there are many options for environments, we highly recommand using `VirtualEnv <https://virtualenv.pypa.io/en/latest/>`_. Start by installing it and creating a directory where all your environments will be saved:
+
+.. code-block:: bash
+
+    sudo pip install virtualenvwrapper
```

Comment:
it's possible this line should now be pip3 but I'm not 100% sure

Reply:
With the previous step (Python) I believe `pip` refers to `pip3`, right?

Reply:
Should we use `pip3` just in case?

Reply:
So I checked and all my `pip` refer to the same version (I don't know why it's the `python3.8` one but everything works fine) so I guess we can keep as is.

```
[master] poic2312@dinf-0051-34b:~/Documents/projects/structure_tensors/code$ pip3.10 --version
pip 23.1.2 from /usr/local/lib/python3.8/dist-packages/pip (python 3.8)
[master] poic2312@dinf-0051-34b:~/Documents/projects/structure_tensors/code$ pip3 --version
pip 23.1.2 from /usr/local/lib/python3.8/dist-packages/pip (python 3.8)
[master] poic2312@dinf-0051-34b:~/Documents/projects/structure_tensors/code$ pip --version
pip 23.1.2 from /usr/local/lib/python3.8/dist-packages/pip (python 3.8)

```